index.html

```diff
@@ -382,10 +382,9 @@ <h2>The <dfn>cancelIdleCallback()</dfn> method</h2>
     </section>
     <section data-dfn-for="IdleDeadline">
       <h2>The <dfn>IdleDeadline</dfn> interface</h2>
-      <p>Each <a>IdleDeadline</a> has an associated <dfn>time</dfn>
-      which holds a {{DOMHighResTimeStamp}} representing the absolute time in
-      milliseconds of the deadline. This MUST be populated when the
-      <a>IdleDeadline</a> is created.</p>
+      <p>Each <a>IdleDeadline</a> has an associated <dfn>time getter</dfn>, an algorithm that
```

Comment:
You should be able to do: 

```suggestion
      <p>Each {{IdleDeadline}} has an associated <dfn>time getter</dfn>, an algorithm that
```

Globally